externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/SparkSQLEngine.scala

```diff
@@ -171,6 +170,11 @@ object SparkSQLEngine extends Logging {
       kyuubiConf.setIfMissing(FRONTEND_CONNECTION_URL_USE_HOSTNAME, false)
     }
 
+    // set web ui port 0 when no deploy k8s cluster mode
```

Comment:
The comment is redundant since the code is clear enough to explain what we are doing here, the really helpful comments should explain WHY we do that.